src/scripts/main.js

```diff
@@ -1,3 +1,194 @@
+form.appendChild(positionLabel);
+
+const ageLabel = document.createElement('label');
+
+ageLabel.innerHTML =
+  'Age: <input name="age" type="number" data-qa="age" required>';
+form.appendChild(ageLabel);
+
+const salaryLabel = document.createElement('label');
+
+salaryLabel.innerHTML =
+  'Salary: <input name="salary" type="number" data-qa="salary" required>';
+form.appendChild(salaryLabel);
+
+const officeSelectLabel = document.createElement('label');
+
+officeSelectLabel.innerHTML = 'Office: ';
+
+const officeSelect = document.createElement('select');
+
+officeSelect.name = 'office';
+officeSelect.setAttribute('data-qa', 'office');
+
+const cities = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+cities.forEach((city) => {
+  const option = document.createElement('option');
+
+  option.value = city;
+  option.textContent = city;
+  officeSelect.appendChild(option);
+});
+
+officeSelectLabel.appendChild(officeSelect);
+form.appendChild(officeSelectLabel);
+
+const button = document.createElement('button');
+
+button.type = 'submit';
+button.textContent = 'Save to table';
+form.appendChild(button);
+
+document.body.appendChild(form);
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  validateForm();
+});
+
+function validateForm() {
+  const nameEmployee = form.elements.name.value;
+  const position = form.elements.position.value;
+  const office = form.elements.office.value;
+  const age = parseInt(form.elements.age.value, 10);
+  const salary = parseFloat(form.elements.salary.value);
+
+  if (nameEmployee.length < 4) {
+    showNotification(
+      'Error',
+      'Name must be at least 4 characters long.',
+      'error',
+    );
+
+    return;
+  }
+
+  if (!position) {
+    showNotification('Error', 'Please choose a valid position.', 'error');
+
+    return;
+  }
+
+  if (age < 18 || age > 90) {
+    showNotification('Error', 'Age must be between 18 and 90.', 'error');
+
+    return;
+  }
+
+  if (isNaN(salary) || salary <= 0) {
+    showNotification('Error', 'Salary must be a positive number.', 'error');
+
+    return;
+  }
+
+  addEmployeeToTable(nameEmployee, position, office, age, salary);
+  showNotification('Success', 'Employee added successfully!', 'success');
+}
+
+function showNotification(title, message, type) {
+  const notification = document.createElement('div');
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
+  notification.setAttribute('data-qa', 'notification');
+
+  const titleElement = document.createElement('span');
+
+  titleElement.classList.add('title');
+  titleElement.textContent = title;
+
+  const messageElement = document.createElement('span');
+
+  messageElement.textContent = message;
+
+  notification.appendChild(titleElement);
+  notification.appendChild(messageElement);
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
+  }, 1000);
+}
```

Comment:
This notification creation code could be simplified and made more readable by using a template literal for the entire notification instead of creating and appending each element separately. This will make your code cleaner and easier to understand.